packages/block-editor/src/components/list-view/utils.js

```diff
@@ -13,8 +13,19 @@ export const getBlockPositionDescription = ( position, siblingCount, level ) =>
 		level
 	);
 
-export const getBlockPropertiesDescription = ( isLocked ) =>
-	isLocked ? __( 'This block is locked.' ) : '';
+export const getBlockPropertiesDescription = ( blockInformation, isLocked ) =>
+	[
+		blockInformation?.positionLabel
+			? `${ sprintf(
+					// translators: 1: Position of selected block, e.g. "Sticky" or "Fixed".
+					__( 'Position: %1$s' ),
+					blockInformation.positionLabel
```

Comment:
```suggestion
					// translators: %s: Position of selected block, e.g. "Sticky" or "Fixed".
					__( 'Position: %s' ),
					blockInformation.positionLabel
```

Nit: This was the case with the original code, but since there is only one placeholder, I don't think the number is necessary.